docs/admin/kubelet-tls-bootstrapping.md

```diff
@@ -17,11 +17,19 @@ and progress on the feature is being tracked as [feature #43](https://github.com
 
 ## kube-apiserver configuration
 
-You must provide a token file which specifies at least one "bootstrap token" assigned to a kubelet bootstrap-specific group.
+The API server should be configured with an [authenticator](/docs/admin/authentication.md) that can authenticate tokens as a user in the `system:bootstrappers` group.
```

Comment:
We don't use the .md extension in links, so the authenticator URL should be: `/docs/admin/authentication/`